NBXplorer/Properties/launchSettings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "iisSettings": {
```

Comment:
I thik you can remove this

Reply:
Fair enough, I wasn't sure if that's needed for some scenarios. I deleted it.